mlos_bench/mlos_bench/environments/status.py

```diff
@@ -19,6 +22,21 @@ class Status(enum.Enum):
     FAILED = 6
     TIMED_OUT = 7
 
+    @staticmethod
+    def from_str(status_str: str) -> "Status":
```

Comment:
Previously we did not restore `Status` on `Trials` restored from the DB.  This helps do that.

Reply:
Should maybe check to see how we were doing that in the `TrialData` class.

Reply:
Did that.